test/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.t.sol

```diff
@@ -122,19 +126,17 @@ contract BancorVirtualSupplyBondingCurveFundingManagerTest is ModuleTest {
     // Events
     event TransferOrchestratorToken(address indexed to, uint indexed amount);
 
-    function setUp() public {
+    function setUp() public virtual {
         // Deploy contracts
-        IBancorVirtualSupplyBondingCurveFundingManager.IssuanceToken memory
-            issuanceToken;
         IBancorVirtualSupplyBondingCurveFundingManager.BondingCurveProperties
             memory bc_properties;
 
         BancorFormula bancorFormula = new BancorFormula();
         formula = address(bancorFormula);
 
-        issuanceToken.name = bytes32(abi.encodePacked(NAME));
-        issuanceToken.symbol = bytes32(abi.encodePacked(SYMBOL));
-        issuanceToken.decimals = uint8(DECIMALS);
+        issuanceToken = new ERC20IssuanceMock();
+        issuanceToken.init(NAME, SYMBOL, type(uint).max, DECIMALS);
+        issuanceToken.setMinter(address(bondingCurveFundingManager));
```

Comment:
I think it actually sets it to 0 the first time since bondingCurveFundingManager is declared but not initialized at that point. I've removed it. Although setting the minter to 0 on setup would indeed be the way to prevent the owner from minting straight away.